### DISCUSSION TITLE
How to set the precision of net speed

### DISCUSSION THREAD

**Discussion by yejianfengblue** [qtile/qtile#3486] [Q&A] [answered]
Below config shows net download speed like 1234.56kB. How to remove the decimal part to be 1234kB or 1235kB?

```python
widget.Net(interface="enp4s0", format='{down}', prefix='k'),
```
The string format `widget.CPU(format='{load_percent:.0f}%'),` works in CPU but not work in Net `widget.Net(interface="enp4s0", format='{down:.0f}', prefix='k'),`

**Comment by elParaguayo** [comment, marked as answer]
Short answer: you can't.

Longer answer: The formatting is hard coded in the widget at the moment as it combines the speed with the unit. The code would need to be reworked to fix this (Pull Requests welcome!)
